benthomasson/eda/tests/unit/requirements.txt

```diff
@@ -0,0 +1,4 @@
+asyncio
```

Comment:
I think is better to have a `test_requirements.txt` file in `/tests` folder because are needed as well for integration tests. 
Ansible-galaxy doesn't take care of this, I had to follow the same approach for collection dependencies here: https://github.com/benthomasson/benthomasson-eda/pull/9